commit 3ce6cdd915c77ba0adee54d0243a52839d846462 (HEAD -> patch-1, origin/patch-1)
Author: Raphael Nestler <raphael.nestler@gmail.com>
Date:   Wed Mar 25 23:53:35 2026 +0100

    Add www/* and sql/* to package-data

    Otherwise it won't be packaged with `python -m build` from https://github.com/pypa/build

diff --git a/pyproject.toml b/pyproject.toml
index e34b715..db2fe32 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -114,7 +114,7 @@ ignore = [


 [tool.setuptools.package-data]
-"*" = ["*.conf"]
+"*" = ["*.conf", "www/*", "sql/*"]


 [tool.setuptools_scm]
